ddtrace/contrib/grpc/aio_server_interceptor.py

```diff
@@ -207,16 +220,34 @@ async def unary_unary(self, request, context):
         return await _wrap_aio_unary_response(self.__wrapped__.unary_unary, request, context, span)
 
     async def unary_stream(self, request, context):
-        # type: (RequestType, aio.ServicerContext) -> ResponseIterableType
+        # type: (RequestType, aio.ServicerContext) -> ResponseType
         span = _create_span(self._pin, self._handler_call_details, constants.GRPC_METHOD_KIND_SERVER_STREAMING)
-        async for response in _wrap_aio_stream_response(self.__wrapped__.unary_stream, request, context, span):
-            yield response
+        return await _wrap_aio_unary_response(self.__wrapped__.unary_stream, request, context, span)
```

Comment:
Hi, I found this while understanding the inteceptor solution and I found this one wierd, isnt `unary_stream` a stream response?